libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/MatrixTimelineDiffProcessor.kt

```diff
@@ -30,20 +29,16 @@ internal class MatrixTimelineDiffProcessor(
     private val timelineItemFactory: MatrixTimelineItemMapper,
 ) {
 
-    private val initLatch = CompletableDeferred<Unit>()
     private val mutex = Mutex()
 
     suspend fun postItems(items: List<TimelineItem>) {
         updateTimelineItems {
             val mappedItems = items.map { it.asMatrixTimelineItem() }
-            addAll(mappedItems)
+            addAll(0, mappedItems)
```

Comment:
Is this 0 here needed? Wouldn't this callback only be called the 1st time we receive items?

Reply:
Nope because I'm now splitting the initialItems in small chunks to post items asap if you have still paginated a lot manually in the room : 
https://github.com/vector-im/element-x-android/blob/94cb694a0dcf47af66a742bdb76cd68a57e3c5a4/libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/RustMatrixTimeline.kt#L89C2-L90C48